filters/ThirdParty/filter_216_PolskiFiltr/metadata.json

```diff
@@ -1,7 +1,7 @@
 {
   "filterId": 216,
   "name": "Official Polish filters for AdBlock, uBlock Origin & AdGuard",
-  "description": "Oficjalne polskie filtry do AdBlocka, uBlocka Origin i AdGuarda",
+  "description": "Polish supplement for EasyList",
```

Comment:
According to your description changes, there is no dot `.` at the end before the second `"`. Is that how it should be?
In most descriptions, I see a dot at the end, and in some I don't.